source/Halibut.Tests/PollingServiceTimeoutsFixture.cs

```diff
@@ -100,9 +100,8 @@ await using (var clientAndService = await clientAndServiceTestCase.CreateTestCas
 
                 Wait.UntilActionSucceeds(() =>
                 {
-                    // Leaving these asserts as when cooperative cancellation is supported it should cause them to fail at which point they can be fixed to assert cancellation to the socket works as expected.
-                    connectionsObserver.ConnectionClosedCount.Should().Be(0, "Cancelling the PendingRequest does not cause the TCP Connection to be cancelled to stop the in-flight request");
-                    connectionsObserver.ConnectionAcceptedCount.Should().Be(1, "The Service won't have reconnected after the request was cancelled");
+                    connectionsObserver.ConnectionClosedCount.Should().Be(1, "Cancelling the PendingRequest does not cause the TCP Connection to be cancelled to stop the in-flight request");
```

Comment:
Does the comment here (and the line below) still apply?

Reply:
Ahh good spot, I haven't updated the because 🙇